wazo_confgend/cache.py

```diff
@@ -20,7 +18,6 @@ def put(self, key, value):
 
 
 class FileCache(Cache):
-
     def __init__(self, basedir):
         super(FileCache, self).__init__()
```

Comment:
`super` calls no longer need the class and `self` so long as the direct parent is the target. So can be changed to `super().__init__()`